Assets/PatchKit Patcher/Scripts/AppData/Remote/RemoteMetaData.cs

```diff
@@ -9,6 +9,8 @@ public class RemoteMetaData : IRemoteMetaData
     {
         private static readonly DebugLogger DebugLogger = new DebugLogger(typeof(RemoteMetaData));
 
+        private const string KeysUrlEnvironmentVariable = "PK_PATCHER_KEYS_URL";
```

Comment:
Are we using more environment variables in the patcher? If yes, these should be listed in a single place. RemoteMetaData does not look like a good place for it.